Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -459,6 +459,14 @@ def audit_deps
   end
 
   def audit_conflicts
+    if formula.name.include?("@")
```

Comment:
might want to put this in `formula.rb` behind a method like `is_version_formula?` to avoid hard coding the "@" character as meaning versioned and hard coding versioned as must have "@".